src/Neo/ProtocolSettings.cs

```diff
@@ -143,7 +179,9 @@ public static ProtocolSettings Load(Stream stream)
         /// <returns>The loaded <see cref="ProtocolSettings"/>.</returns>
         public static ProtocolSettings Load(string path)
         {
-            if (!File.Exists(path))
+            path = FindFile(path, Environment.CurrentDirectory);
```

Comment:
My primary concern here is that this code should be compatible with the old behaviour of `neo-cli` binary for production usage (default `config.json` file from the `neo-cli` binary dir will be used). From what I see, it's OK, but we'd better recheck.